src/CADL.Extension/Emitter.Csharp/src/lib/ServiceAuthentication.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we make the first letter small case?

Reply:
some of our files start with an upper case letter, the others start with a lower case letter. Could we align?

Reply:
The TS official code style doesn't mention file name case. It only says the class should be upper camel case. However, I found a code guide of a framework, in which the file name should be uppder camel case: https://www.itwinjs.org/learning/guidelines/typescript-coding-guidelines/#:~:text=Files%201%20Use%20the%20.ts%20file%20extension%20for,files%202%20TypeScript%20file%20names%20should%20be%20PascalCase

![image](https://user-images.githubusercontent.com/4516/223905847-f675c8ff-a7e8-4484-b93e-245b460bb1fa.png)

Reply:
consistent naming convention, the File name is cameCase. Update the file names.